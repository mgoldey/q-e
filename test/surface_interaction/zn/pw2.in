&CONTROL
  calculation  = "scf",
  prefix       = "left",
  outdir       = "./out",
  pseudo_dir   = "../../pseudo",
  verbosity = "high" 
  wf_collect = .true.
/
&SYSTEM
  nosym = .true.
  nbnd = 14
  ibrav = 1,
  a = 12,
  nat = 2,
  ntyp = 1,
  ecutwfc = 50,
  tot_charge = +1
  nspin = 2,
  tot_magnetization = 1
/
&ELECTRONS
 ! startingpot = 'file'
 ! startingwfc = 'file'
  conv_thr    = 1.D-7,
  mixing_beta = 0.30,
/
EPCDFT
1 1.D+2 1.D-2 100 .true.
delta_charge 1 1 2 2 0.0 0.0

K_POINTS {Gamma}

ATOMIC_SPECIES
Zn 0.0 Zn.pbe-mt_fhi.UPF
ATOMIC_POSITIONS (angstrom)
Zn 5 5 4
Zn 5 5 2
