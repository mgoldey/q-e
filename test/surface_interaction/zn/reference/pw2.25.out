
     Program PWSCF v.5.1.1 starts on 10May2016 at 14:22:41 

     This program is part of the open-source Quantum ESPRESSO suite
     for quantum simulation of materials; please cite
         "P. Giannozzi et al., J. Phys.:Condens. Matter 21 395502 (2009);
          URL http://www.quantum-espresso.org", 
     in publications or presentations arising from this work. More details at
     http://www.quantum-espresso.org/quote

     Parallel version (MPI), running on     4 processors
     R & G space division:  proc/nbgrp/npool/nimage =       4
     Waiting for input...
     Reading input from standard input
 Reading EPCDFT CARD. Printing based upon belief that input should be reproducable from output.
     Reading   1 constraint(s); tolerance:  100.000000 max change in field 0.01
     epcdft_update_intrvl: 100
     epcdft with surface: T
          constraint           type   A   A   D   D      T      L
                   1  delta_charge    1   1   2   2 0.0000 0.0000

     Current dimensions of program PWSCF are:
     Max number of different atomic species (ntypx) = 10
     Max number of k-points (npk) =  40000
     Max angular momentum in pseudopotentials (lmaxx) =  3
               file Zn.pbe-mt_fhi.UPF: wavefunction(s)  4p 4f renormalized

     gamma-point specific algorithms are used
     Message from routine setup:
     the system is metallic, specify occupations

     Subspace diagonalization in iterative solution of the eigenvalue problem:
     a serial algorithm will be used


     Parallelization info
     --------------------
     sticks:   dense  smooth     PW     G-vecs:    dense   smooth      PW
     Min        2047    2047    512               139165   139165   17446
     Max        2052    2052    514               139168   139168   17450
     Sum        8197    8197   2053               556667   556667   69791
     Tot        4099    4099   1027

     Generating pointlists ...
     new r_m :   0.0688 (alat units)  1.5590 (a.u.) for type    1


     bravais-lattice index     =            1
     lattice parameter (alat)  =      22.6767  a.u.
     unit-cell volume          =   11661.1222 (a.u.)^3
     number of atoms/cell      =            2
     number of atomic types    =            1
     number of electrons       =        23.00 (up:  12.00, down:  11.00)
     number of Kohn-Sham states=           14
     kinetic-energy cutoff     =      50.0000  Ry
     charge density cutoff     =     200.0000  Ry
     convergence threshold     =      1.0E-07
     mixing beta               =       0.3000
     number of iterations used =            8  plain     mixing
     Exchange-correlation      = PBE ( 1  4  3  4 0 0)

     celldm(1)=  22.676714  celldm(2)=   0.000000  celldm(3)=   0.000000
     celldm(4)=   0.000000  celldm(5)=   0.000000  celldm(6)=   0.000000

     crystal axes: (cart. coord. in units of alat)
               a(1) = (   1.000000   0.000000   0.000000 )  
               a(2) = (   0.000000   1.000000   0.000000 )  
               a(3) = (   0.000000   0.000000   1.000000 )  

     reciprocal axes: (cart. coord. in units 2 pi/alat)
               b(1) = (  1.000000  0.000000  0.000000 )  
               b(2) = (  0.000000  1.000000  0.000000 )  
               b(3) = (  0.000000  0.000000  1.000000 )  


     PseudoPot. # 1 for Zn read from file:
     ../../pseudo/Zn.pbe-mt_fhi.UPF
     MD5 check sum: 5596c9f2b89b9f7367257e7ea958b26e
     Pseudo is Norm-conserving, Zval = 12.0
     Generated using FHI98PP, converted with fhi2upf.x v.5.0.2
     Using radial grid of  527 points,  3 beta functions with: 
                l(1) =   1
                l(2) =   2
                l(3) =   3

     atomic species   valence    mass     pseudopotential
        Zn            12.00    65.39000     Zn( 1.00)

     Starting magnetic structure 
     atomic species   magnetization
        Zn           0.000

     No symmetry found


                                    s                        frac. trans.

      isym =  1     identity                                     

 cryst.   s( 1) = (     1          0          0      )
                  (     0          1          0      )
                  (     0          0          1      )

 cart.    s( 1) = (  1.0000000  0.0000000  0.0000000 )
                  (  0.0000000  1.0000000  0.0000000 )
                  (  0.0000000  0.0000000  1.0000000 )


     point group C_1 (1)    
     there are  1 classes
     the character table:

       E    
A      1.00

     the symmetry operations in each class:
     E        1

   Cartesian axes

     site n.     atom                  positions (alat units)
         1           Zn  tau(   1) = (   0.4166667   0.4166667   0.3541667  )
         2           Zn  tau(   2) = (   0.4166667   0.4166667   0.1875000  )

   Crystallographic axes

     site n.     atom                  positions (cryst. coord.)
         1           Zn  tau(   1) = (  0.4166667  0.4166667  0.3541667  )
         2           Zn  tau(   2) = (  0.4166667  0.4166667  0.1875000  )

     number of k points=     2
                       cart. coord. in units 2pi/alat
        k(    1) = (   0.0000000   0.0000000   0.0000000), wk =   1.0000000
        k(    2) = (   0.0000000   0.0000000   0.0000000), wk =   1.0000000

                       cryst. coord.
        k(    1) = (   0.0000000   0.0000000   0.0000000), wk =   1.0000000
        k(    2) = (   0.0000000   0.0000000   0.0000000), wk =   1.0000000

     Dense  grid:   278334 G-vectors     FFT dimensions: ( 108, 108, 108)

     Largest allocated arrays     est. size (Mb)     dimensions
        Kohn-Sham Wavefunctions         1.86 Mb     (    8725,   14)
        NL pseudopotentials             3.99 Mb     (    8725,   30)
        Each V/rho on FFT grid          9.61 Mb     (  314928,   2)
        Each G-vector array             0.53 Mb     (   69584)
        G-vector shells                 0.02 Mb     (    2154)
     Largest temporary arrays     est. size (Mb)     dimensions
        Auxiliary wavefunctions         3.73 Mb     (    8725,   56)
        Each subspace H/S matrix        0.02 Mb     (      56,   56)
        Each <psi_i|beta_j> matrix      0.00 Mb     (      30,   14)
        Arrays for rho mixing          38.44 Mb     (  314928,    8)

     Initial potential from superposition of free atoms
     Check: negative starting charge=(component1):   -0.041632
     Check: negative starting charge=(component2):   -0.041632

     starting charge   23.99975, renormalised to   23.00000

     negative rho (up, down):  3.990E-02 3.990E-02
     
     Adding image charge field due to neutral metal slab at origin in XY plane.
     Including monopole and dipole terms.
     Quadrupole not implemented yet.
     System should not overlap with cell edges.
     

     reference position (x0):         9.44863066    9.44863066    6.14160993 bohr
     Total charge   1.0000
     Total dipole moment   0.0004   0.0004  -0.0001 au (Ha),   0.0010   0.0010  -0.0003 Debye


     First surface image interaction energy:-0.170E+00[Ry]


     reference position (x0):         9.44863066    9.44863066    6.14160993 bohr
     Total charge   1.0000
     Total dipole moment   0.0004   0.0004  -0.0001 au (Ha),   0.0010   0.0010  -0.0003 Debye

     Starting wfc are   32 randomized atomic wfcs

     total cpu time spent up to now is        3.5 secs

     per-process dynamical memory:    78.4 Mb

     Self-consistent Calculation

     iteration #  1     ecut=    50.00 Ry     beta=0.30
     Davidson diagonalization with overlap
     ethr =  1.00E-02,  avg # of iterations =  4.0

     negative rho (up, down):  2.720E-02 2.748E-02

     Magnetic moment per site:
     atom:    1    charge:    9.3467    magn:    0.0396    constr:    0.0000
     atom:    2    charge:    9.3284    magn:    0.0364    constr:    0.0000

     total cpu time spent up to now is        6.8 secs

     total energy              =    -217.96561851 Ry
     Harris-Foulkes estimate   =    -218.17917185 Ry
     estimated scf accuracy    <       0.55164607 Ry

     total magnetization       =     1.00 Bohr mag/cell
     absolute magnetization    =     1.00 Bohr mag/cell

     iteration #  2     ecut=    50.00 Ry     beta=0.30
     Davidson diagonalization with overlap
     ethr =  2.40E-03,  avg # of iterations =  1.5

     negative rho (up, down):  1.242E-02 1.282E-02

     Magnetic moment per site:
     atom:    1    charge:    9.4675    magn:    0.0777    constr:    0.0000
     atom:    2    charge:    9.4502    magn:    0.0751    constr:    0.0000

     total cpu time spent up to now is        9.5 secs

     total energy              =    -217.96173426 Ry
     Harris-Foulkes estimate   =    -218.01222521 Ry
     estimated scf accuracy    <       0.11892136 Ry

     total magnetization       =     1.00 Bohr mag/cell
     absolute magnetization    =     1.01 Bohr mag/cell

     iteration #  3     ecut=    50.00 Ry     beta=0.30
     Davidson diagonalization with overlap
     ethr =  5.17E-04,  avg # of iterations =  2.0

     negative rho (up, down):  6.453E-03 6.822E-03

     Magnetic moment per site:
     atom:    1    charge:    9.4594    magn:    0.0858    constr:    0.0000
     atom:    2    charge:    9.4525    magn:    0.0850    constr:    0.0000

     total cpu time spent up to now is       12.5 secs

     total energy              =    -217.99065616 Ry
     Harris-Foulkes estimate   =    -217.98872464 Ry
     estimated scf accuracy    <       0.00980968 Ry

     total magnetization       =     1.00 Bohr mag/cell
     absolute magnetization    =     1.04 Bohr mag/cell

     iteration #  4     ecut=    50.00 Ry     beta=0.30
     Davidson diagonalization with overlap
     ethr =  4.27E-05,  avg # of iterations =  2.0

     negative rho (up, down):  3.889E-05 4.065E-05

     Magnetic moment per site:
     atom:    1    charge:    9.4791    magn:    0.0975    constr:    0.0000
     atom:    2    charge:    9.4733    magn:    0.0986    constr:    0.0000

     total cpu time spent up to now is       15.3 secs

     total energy              =    -217.99097500 Ry
     Harris-Foulkes estimate   =    -217.99137915 Ry
     estimated scf accuracy    <       0.00279132 Ry

     total magnetization       =     1.00 Bohr mag/cell
     absolute magnetization    =     1.05 Bohr mag/cell

     iteration #  5     ecut=    50.00 Ry     beta=0.30
     Davidson diagonalization with overlap
     ethr =  1.21E-05,  avg # of iterations =  2.5

     negative rho (up, down):  1.978E-05 2.052E-05

     Magnetic moment per site:
     atom:    1    charge:    9.4719    magn:    0.0951    constr:    0.0000
     atom:    2    charge:    9.4661    magn:    0.0963    constr:    0.0000

     total cpu time spent up to now is       18.4 secs

     total energy              =    -217.99221325 Ry
     Harris-Foulkes estimate   =    -217.99193352 Ry
     estimated scf accuracy    <       0.00050395 Ry

     total magnetization       =     1.00 Bohr mag/cell
     absolute magnetization    =     1.08 Bohr mag/cell

     iteration #  6     ecut=    50.00 Ry     beta=0.30
     Davidson diagonalization with overlap
     ethr =  2.19E-06,  avg # of iterations =  6.0

     Magnetic moment per site:
     atom:    1    charge:    9.4691    magn:    0.0930    constr:    0.0000
     atom:    2    charge:    9.4631    magn:    0.0944    constr:    0.0000

     total cpu time spent up to now is       21.6 secs

     total energy              =    -217.99209395 Ry
     Harris-Foulkes estimate   =    -217.99192179 Ry
     estimated scf accuracy    <       0.00003509 Ry

     total magnetization       =     1.00 Bohr mag/cell
     absolute magnetization    =     1.08 Bohr mag/cell

     iteration #  7     ecut=    50.00 Ry     beta=0.30
     Davidson diagonalization with overlap
     ethr =  1.53E-07,  avg # of iterations =  3.0

     Magnetic moment per site:
     atom:    1    charge:    9.4682    magn:    0.0922    constr:    0.0000
     atom:    2    charge:    9.4623    magn:    0.0936    constr:    0.0000

     total cpu time spent up to now is       24.8 secs

     total energy              =    -217.99203727 Ry
     Harris-Foulkes estimate   =    -217.99202064 Ry
     estimated scf accuracy    <       0.00000422 Ry

     total magnetization       =     1.00 Bohr mag/cell
     absolute magnetization    =     1.09 Bohr mag/cell

     iteration #  8     ecut=    50.00 Ry     beta=0.30
     Davidson diagonalization with overlap
     ethr =  1.83E-08,  avg # of iterations =  2.5

     Magnetic moment per site:
     atom:    1    charge:    9.4688    magn:    0.0920    constr:    0.0000
     atom:    2    charge:    9.4628    magn:    0.0934    constr:    0.0000

     total cpu time spent up to now is       27.9 secs

     total energy              =    -217.99199765 Ry
     Harris-Foulkes estimate   =    -217.99205530 Ry
     estimated scf accuracy    <       0.00000402 Ry

     total magnetization       =     1.00 Bohr mag/cell
     absolute magnetization    =     1.09 Bohr mag/cell

     iteration #  9     ecut=    50.00 Ry     beta=0.30
     Davidson diagonalization with overlap
     ethr =  1.75E-08,  avg # of iterations =  2.0

     Magnetic moment per site:
     atom:    1    charge:    9.4686    magn:    0.0918    constr:    0.0000
     atom:    2    charge:    9.4627    magn:    0.0932    constr:    0.0000

     total cpu time spent up to now is       31.2 secs

     total energy              =    -217.99203567 Ry
     Harris-Foulkes estimate   =    -217.99203662 Ry
     estimated scf accuracy    <       0.00000015 Ry

     total magnetization       =     1.00 Bohr mag/cell
     absolute magnetization    =     1.09 Bohr mag/cell

     iteration # 10     ecut=    50.00 Ry     beta=0.30
     Davidson diagonalization with overlap
     ethr =  6.32E-10,  avg # of iterations =  2.5

     Magnetic moment per site:
     atom:    1    charge:    9.4688    magn:    0.0918    constr:    0.0000
     atom:    2    charge:    9.4628    magn:    0.0933    constr:    0.0000

     total cpu time spent up to now is       34.3 secs

     total energy              =    -217.99203360 Ry
     Harris-Foulkes estimate   =    -217.99204191 Ry
     estimated scf accuracy    <       0.00000015 Ry

     total magnetization       =     1.00 Bohr mag/cell
     absolute magnetization    =     1.09 Bohr mag/cell

     iteration # 11     ecut=    50.00 Ry     beta=0.30
     Davidson diagonalization with overlap
     ethr =  6.32E-10,  avg # of iterations =  2.0

     reference position (x0):         9.44863066    9.44863066    6.14160993 bohr
     Total charge   1.0000
     Total dipole moment  -0.0004  -0.0004  -0.5040 au (Ha),  -0.0010  -0.0010  -1.2810 Debye

       I         D           A         Val      Target            Str
       1 0.112E+01   0.100E+01   0.124E+00   0.000E+00   0.000000E+00
     Image interaction energy: -0.169E+00 [Ry] Change:  0.999E-03 [Ry] Tolerance:  0.100E-04 [Ry]

     Magnetic moment per site:
     atom:    1    charge:    9.4688    magn:    0.0918    constr:    0.0000
     atom:    2    charge:    9.4628    magn:    0.0933    constr:    0.0000

     total cpu time spent up to now is       37.8 secs

     total energy              =    -217.99204246 Ry
     Harris-Foulkes estimate   =    -217.99203858 Ry
     estimated scf accuracy    <          4.5E-09 Ry

     total magnetization       =     1.00 Bohr mag/cell
     absolute magnetization    =     1.09 Bohr mag/cell

     iteration # 12     ecut=    50.00 Ry     beta=0.30
     Davidson diagonalization with overlap
     ethr =  1.98E-11,  avg # of iterations =  2.0

     reference position (x0):         9.44863066    9.44863066    6.14160993 bohr
     Total charge   1.0000
     Total dipole moment  -0.0004  -0.0004  -0.5040 au (Ha),  -0.0010  -0.0010  -1.2810 Debye

       I         D           A         Val      Target            Str
       1 0.112E+01   0.100E+01   0.124E+00   0.000E+00   0.000000E+00
     Image interaction energy: -0.169E+00 [Ry] Change:  0.416E-07 [Ry] Tolerance:  0.100E-04 [Ry]

     reference position (x0):         9.44863066    9.44863066    6.14160993 bohr
     Total charge   1.0000
     Total dipole moment  -0.0004  -0.0004  -0.5040 au (Ha),  -0.0010  -0.0010  -1.2810 Debye


     Magnetic moment per site:
     atom:    1    charge:    9.4688    magn:    0.0919    constr:    0.0000
     atom:    2    charge:    9.4628    magn:    0.0933    constr:    0.0000

     total cpu time spent up to now is       40.4 secs

     End of self-consistent calculation

 ------ SPIN UP ------------


          k = 0.0000 0.0000 0.0000 ( 34896 PWs)   bands (ev):

   -13.2254   -12.5172   -12.5172   -11.7538   -11.7499   -11.5133   -11.4968   -11.2794
   -11.2794   -11.2335    -9.2998    -5.4029    -2.6963    -1.7772
     occupation numbers 
     1.0000   1.0000   1.0000   1.0000   1.0000   1.0000   1.0000   1.0000
     1.0000   1.0000   1.0000   1.0000   0.0000   0.0000

 ------ SPIN DOWN ----------


          k = 0.0000 0.0000 0.0000 ( 34896 PWs)   bands (ev):

   -13.0914   -12.4075   -12.4075   -11.6777   -11.6738   -11.4333   -11.4170   -11.1255
   -11.1254   -11.0467    -9.0544    -4.4798    -2.3436    -2.3436
     occupation numbers 
     1.0000   1.0000   1.0000   1.0000   1.0000   1.0000   1.0000   1.0000
     1.0000   1.0000   1.0000   0.0000   0.0000   0.0000

     highest occupied level (ev):    -5.4029

     reference position (x0):         9.44863066    9.44863066    6.14160993 bohr
     Total charge   1.0000
     Total dipole moment  -0.0004  -0.0004  -0.5040 au (Ha),  -0.0010  -0.0010  -1.2810 Debye

       I         D           A         Val      Target            Str
       1 0.112E+01   0.100E+01   0.124E+00   0.000E+00   0.000000E+00
     Image interaction energy: -0.169E+00 [Ry] Change:  0.416E-07 [Ry] Tolerance:  0.100E-04 [Ry]

!    total energy              =    -217.99203082 Ry
     Harris-Foulkes estimate   =    -217.99203858 Ry
     estimated scf accuracy    <          4.7E-09 Ry

     The total energy is the sum of the following terms:

     one-electron contribution =    -399.98980343 Ry
     hartree contribution      =     209.58109527 Ry
     xc contribution           =     -32.48498880 Ry
     ewald contribution        =       4.90223735 Ry
     CDFT correction           =      -0.00000000 Ry
     scf correction            =      -0.00057121 Ry

     total magnetization       =     1.00 Bohr mag/cell
     absolute magnetization    =     1.09 Bohr mag/cell

     convergence has been achieved in  12 iterations

     Writing output data file left.save

     init_run     :      3.35s CPU      3.48s WALL (       1 calls)
     electrons    :     36.02s CPU     37.52s WALL (       1 calls)

     Called by init_run:
     wfcinit      :      1.17s CPU      1.18s WALL (       1 calls)
     potinit      :      1.59s CPU      1.69s WALL (       1 calls)

     Called by electrons:
     c_bands      :     16.89s CPU     17.61s WALL (      12 calls)
     sum_band     :      4.63s CPU      4.78s WALL (      12 calls)
     v_of_rho     :     11.30s CPU     11.74s WALL (      12 calls)
     v_h          :      0.38s CPU      0.40s WALL (      12 calls)
     v_xc         :     10.92s CPU     11.34s WALL (      12 calls)
     mix_rho      :      1.66s CPU      1.70s WALL (      12 calls)

     Called by c_bands:
     init_us_2    :      0.39s CPU      0.41s WALL (      50 calls)
     regterg      :     16.17s CPU     16.85s WALL (      24 calls)

     Called by sum_band:

     Called by *egterg:
     h_psi        :     15.76s CPU     16.38s WALL (      90 calls)
     g_psi        :      0.07s CPU      0.07s WALL (      64 calls)
     rdiaghg      :      0.04s CPU      0.04s WALL (      88 calls)
     regterg:over :      0.52s CPU      0.54s WALL (      64 calls)
     regterg:upda :      0.33s CPU      0.34s WALL (      64 calls)
     regterg:last :      0.13s CPU      0.14s WALL (      24 calls)

     Called by h_psi:
     h_psi:vloc   :     14.88s CPU     15.48s WALL (      90 calls)
     h_psi:vnl    :      0.79s CPU      0.82s WALL (      90 calls)
     add_vuspsi   :      0.36s CPU      0.37s WALL (      90 calls)

     General routines
     calbec       :      0.44s CPU      0.45s WALL (      90 calls)
     fft          :      6.91s CPU      7.21s WALL (     313 calls)
     fftw         :     14.70s CPU     15.30s WALL (    1146 calls)

     Parallel routines
     fft_scatter  :     10.35s CPU     10.79s WALL (    1459 calls)

     PWSCF        :    39.56s CPU        41.24s WALL


   This run was terminated on:  14:23:22  10May2016            

=------------------------------------------------------------------------------=
   JOB DONE.
=------------------------------------------------------------------------------=
